app/config/dev.json

```diff
@@ -8,18 +7,9 @@
   "PGPORT": 5432,
   "PGDATABASE": "ccbc",
   "PGSCHEMA": "ccbc_public",
-  "HAPPO_API_KEY": "",
-  "HAPPO_API_SECRET": "",
-  "CLIENT_SECRET": "",
   "HOST": "localhost",
-  "OPENSHIFT_APP_NAMESPACE": "",
+  "OPENSHIFT_APP_NAMESPACE": "ff61fb-dev",
   "SESSION_SECRET": "superSecretSession",
-  "NEXT_PUBLIC_GROWTHBOOK_API_KEY": "",
-  "AWS_S3_BUCKET": "",
-  "AWS_S3_REGION": "us-west-2",
-  "AWS_S3_KEY": "",
-  "AWS_S3_SECRET_KEY": "",
-  "AWS_ROLE_ARN": "",
   "SITEMINDER_LOGOUT_URL": "https://logontest7.gov.bc.ca/clp-cgi/logoff.cgi",
   "ENABLE_MOCK_TIME" : true,
   "ENABLE_MOCK_COOKIES" : true
```

Comment:
Do we need anything else than those three variables? The rest is set by the environment variables in the helm chart